Conceptpower+Spring/src/main/java/edu/asu/conceptpower/wrapper/ConceptEntryWrapper.java

```diff
@@ -17,6 +19,8 @@
  * @author Julia Damerow
  * 
  */
+@Component("conceptEntryWrapper")
+@Scope(value = "prototype")
 public class ConceptEntryWrapper implements Serializable {
```

Comment:
Didn't we discuss this? That should be just a simple class, no scope, no component. And then the uri is a property that is set in the wrapper creator.